src/Components/Footer/Footer.js

```diff
@@ -1 +1,240 @@
+            </marquee>
+          </MailBox>
+        </LeftBox>
+        <RightBox>
+          <LinkList>
+            <li>HELP</li>
+            <li>Contact Us</li>
+            <li>FAQ</li>
+            <li>Shipments</li>
+            <li>Payments</li>
+            <li>Track Your Order</li>
+            <li>Return Your Order</li>
+            <li>Subscribe To The Newsletter</li>
+          </LinkList>
+          <LinkList>
+            <li>LEGAL INFO</li>
+            <li>Privacy Policy</li>
+            <li>Terms & Conditions</li>
+            <li>Return Policy</li>
+            <li>Cookie Policy</li>
+            <li>Gift Card</li>
+          </LinkList>
+          <LinkList>
+            <li>FOLLOW US</li>
+            <li>Instagram</li>
+            <li>Facebook</li>
+          </LinkList>
+        </RightBox>
+      </MiddleBox>
+      <BottomBox>
+        <svg viewBox="0 0 1331 160" className="logo">
+          <g>
+            <path d={superfluidNav.logo}></path>
+          </g>
+        </svg>
+        <div>
+          <div className="philosophyBoxes">
+            <div>
+              <svg viewBox="0 0 13 20">
+                <g>
+                  <path d={superfluidFooter.human}></path>
+                </g>
+              </svg>
+              <p>HUMAN</p>
+            </div>
+            <div>
+              <svg viewBox="0 0 21 19">
+                <g>
+                  <path d={superfluidFooter.vegan}></path>
+                </g>
+              </svg>
+              <p>VEGAN</p>
+            </div>
+            <div>
+              <svg viewBox="0 0 19 16">
+                <g>
+                  <path d={superfluidFooter.inclusive}></path>
+                </g>
+              </svg>
+              <p>INCLUSIVE</p>
+            </div>
+          </div>
+          <span>© 2020 Superfluid - All Rights Reserved</span>
+        </div>
+      </BottomBox>
+    </>
+  );
+};
+
+export default Footer;
+
+const TopBox = styled.footer`
+  height: 330px;
+  display: flex;
+  font-size: 35px;
+  font-weight: 800;
+
+  li {
+    width: 33.3%;
+    border: 2px solid black;
+    border-left: none;
+    display: felx;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    font-family: AllianceB;
+
+    &:last-child {
+      border-right: 0;
+    }
+  }
+`;
+
+const MiddleBox = styled.div`
+  height: 375px;
+  padding: 70px 0 70px 47px;
+  display: flex;
+  justify-content: space-between;
+`;
+
+const LeftBox = styled.div`
+  height: 300px;
+  padding-right: 15px;
+  font-size: 35px;
+
+  p {
+    width: 213px;
+    font-family: AllianceB;
+    line-height: 35px;
+    letter-spacing: -1px;
+    margin-bottom: 60px;
+  }
+`;
+
+const MailBox = styled.div`
+  height: 100px;
+
+  form {
+    width: 320px;
+    position: relative;
+
+    input {
+      width: 320px;
+      padding-bottom: 10px;
+      margin-bottom: 30px;
+      border: none;
+      border-bottom: 2px solid black;
```

Comment:
```suggestion
      border: none;
      border-bottom: 2px solid black;
```
border none을 주신 이유가 궁금합니다!

Reply:
input tag여서 border 속성이 디폴트로 지정되어서 none을 주고 bottom만 따로 주어야 border-bottom 만 생기더라구요:)!!